src/modules/localStorage.js

```diff
@@ -107,4 +107,77 @@ export default class toDoStorage {
       );
     }
   }
+
+  markAsReadLocalStorageData(index) {
+    // Responsible to mark todo as read in local storage
+    const currentTodoData = JSON.parse(
+      localStorage.getItem(this.localStorageObjectName)
+    );
+
+    currentTodoData.forEach((element) => {
+      if (parseInt(element.index, 10) === parseInt(index, 10)) {
+        // console.log(element);
+        element.completed = true;
+      }
+    });
+
+    localStorage.setItem(
+      this.localStorageObjectName,
+      JSON.stringify(currentTodoData)
+    );
+  }
+
+  markAsUnreadLocalStorageData(index) {
+    // Responsible to mark as unread todo in local storage
+    const currentTodoData = JSON.parse(
+      localStorage.getItem(this.localStorageObjectName)
+    );
+
+    currentTodoData.forEach((element) => {
+      if (parseInt(element.index, 10) === parseInt(index, 10)) {
+        // console.log(element);
+        element.completed = false;
+      }
+    });
+
+    localStorage.setItem(
+      this.localStorageObjectName,
+      JSON.stringify(currentTodoData)
+    );
+  }
```

Comment:
after creating a local object you wont need to call localStorage getItem and these two functions can be merged into only one that you can call toggleMark or toggleComplete `element.completed = !element.completed`

Reply:
@souad988 I prefer this coding style. Why you are forcing me to follow your style? Is it required for project approval? A reviewer is meant to find out the following key requirements and bugs. Not to follow any specific coding style.

Reply:
- [ ] A code reviewer's role is to guide you to write better code. Code that has a lot of duplication is hard to maintain. It is therefore my humble suggestion that you write DRY code. Please update you code inline with[ this suggestion](https://github.com/Aqurds/t-d-l/pull/3/files#r1232164472) 👍🏾

Reply:
Applied this suggestion. Thanks!